commands/commands_test.go

```diff
@@ -513,37 +513,37 @@ func TestCommandsAPIExecute_NoCommandResults(t *testing.T) {
 	})
 }
 
-func TestAccContext(t *testing.T) {
```

Comment:
and this one?

Reply:
moved to `commands/acceptance/commands_test.go`